### ISSUE TITLE
Issue with pagination

### ISSUE THREAD

**Issue by anuragte88** [Luracast/Laravel-Database#6]
Thanks for creating this wonderful package. I have used this package with Restler 3.0 but got pagination error. So please include illuminate/pagination component in composer.json file.

**Comment by Arul-** [comment]
You have used it laravel 4.2.* components or 5.2.* components?

**Comment by Arul-** [comment]
Also posting the exact error message you are getting and a sample code to reproduce will help

**Comment by anuragte88** [comment]
I have used 5.2.* component with Restler 3.0. Below are the steps to reproduce this error.

1. Create laravel model file. For example I have created Book.php inside app/model directory.
2. I have used this model in Restler api class. When I try to return pagination data in Api call then it show  error "Illuminate\Pagination\Paginator class not found".  
3. I have used simple laravel model method to return pagination data. For example :- Book::paginate(10);
4. So for resolve above issue. I have included "illuminate/pagination" : "5.2.*" component in composer.json file and run composer update command. After this issue resolved.
